__tests__/commands/install/integration.js

```diff
@@ -491,6 +491,19 @@ test.concurrent('install file: dedupe dependencies 2', (): Promise<void> => {
   });
 });
 
+// When local packages are installed from a repo with a lockfile, the multiple packages
+// unpacking in the same location warning should not occur
+test.concurrent('install file: dedupe dependencies 3', (): Promise<void> => {
+  return runInstall({}, 'install-file-dedupe-dependencies-3', (config, reporter, install, getStdout) => {
+    const stdout = getStdout();
+    // Need to check if message is logged, but don't need to check for any specific parameters
+    // so splitting on undefined and testing if all message parts are in stdout
```

Comment:
So the only downside I could see for that, is the test would then expand to cover all the warning, info, and success messages logged when an install is done. In the situation when one of those changes, it might be confusing for a test for file protocol installs to fail? What do you think?